kubetest/anywhere.go

```diff
@@ -267,24 +270,77 @@ func (k *kubernetesAnywhere) DumpClusterLogs(localPath, gcsPath string) error {
 		return nil
 	}
 
-	// the e2e framework in k/k does not support the "kubernetes-anywhere" provider,
-	// while a valid provider is required by the k/k "./cluster/log-dump/log-dump.sh" script
-	// for dumping the logs of the GCE cluster that kubernetes-anywhere creates:
-	//   https://github.com/kubernetes/kubernetes/blob/master/cluster/log-dump/log-dump.sh
-	// this fix is quite messy, but an acceptable workaround until "anywhere.go" is removed completely.
-	//
-	// TODO(neolit123): this workaround can be removed if defaultDumpClusterLogs() is refactored to
-	// not use log-dump.sh.
-	providerKey := "KUBERNETES_PROVIDER"
-	oldValue := os.Getenv(providerKey)
-	if err := os.Setenv(providerKey, "gce"); err != nil {
+	// hardcoding the private key path
+	const privateKeyPath = "/workspace/.ssh/google_compute_engine"
```

Comment:
not sure if this is the key that can be used.

Reply:
Or you can rely on the two envs in https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L526-L543?

Reply:
oh yeah they got copied over to https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_e2e.py#L226-L239... maybe rely on the env instead of a hard-code path?